vmcloak/abstract.py

```diff
@@ -21,9 +22,10 @@
 
 log = logging.getLogger(__name__)
 
-GENISOIMAGE_ERROR = (
-    "Warning: creating filesystem that does not conform to ISO-9660.\n"
-)
+GENISOIMAGE_WARNINGS = [
+    "Warning: creating filesystem that does not conform to ISO-9660.",
+    "Warning: creating filesystem that does not conform to ISO-9660. Warning: creating filesystem with (nonstandard) Joliet extensions but without (standard) Rock Ridge extensions. It is highly recommended to add Rock Ridge",
```

Comment:
Can you split this line over multiple lines so to adhere to pep8 80-char convention etc?